### PR TITLE
Add pod subspecs and component name [ Update IntegrationWithExistingApps.md ]

### PR DESCRIPTION
**Pull request by crazy9code**

### CODE REVIEW

File: docs/IntegrationWithExistingApps.md [hunk 1]
```diff
@@ -439,7 +440,7 @@ import React
 
   let rootView = RCTRootView(
       bundleURL: jsCodeLocation,
-      moduleName: "RNHighScores",
+      moduleName: "MyReactNativeApp",
```
Comment (piotrek1543) [inline docs/IntegrationWithExistingApps.md]: Ok, I agree that there's a bug to fix it, but it's already fixed in another merge request. Compare with: https://github.com/facebook/react-native/pull/15252/files

**Comment by piotrek1543** [comment]
@crazy9code could you change your's MR name to more descriptive? Thanks

**Comment by crazy9code** [comment]
@hramos there is no issue with doc please update

**Comment by hramos** [comment]
@crazy9code as @piotrek1543 suggested earlier, your pull request is missing a description. Generally, we like to know what the motivation for a given change is, so we can review accordingly. Thanks!

**Comment by crazy9code** [comment]
@hramos as @piotrek1543 suggestion, I had to change the pull request description in the third commit and again some check was not successful, then I have seen the details and find this issue related to Xcode version, there is no issue with Doc.

**Comment by hramos** [comment]
Your original post is still showing empty for me.

screen shot 2017-08-17 at 1 06 45 pm

**Pull request closed without merging**